lib/cacheOptimization.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
```

Comment:
Just merge master into ao-master and then merge ao-master into this branch and everything will be as expected.